### DISCUSSION TITLE
Why there is no highlight in tsx code  when uses @rspress/plugin-preview?

### DISCUSSION THREAD

**Discussion by PuffMeow** [web-infra-dev/rspress#1112] [Q&A]
I add the plugin of @rspress/plugin-preview. Then I use code to import my tsx demo component, there is no highlight.
Like this:
`<code src="./Demo.tsx" />`

**Comment by Timeless0911** [comment]
Currently rspress only supports highlighting of code using syntax like below
```
```tsx
```

**Reply by PuffMeow** [reply]
Is there any plans to support it?

**Reply by Timeless0911** [reply]
You can make an issue with PR Welcome labeled to help us tracking on it, I think it should be supported but maybe not ASAP as you expected.
